northstar/src/npk/npk.rs

```diff
@@ -559,11 +560,8 @@ pub fn generate_key(name: &str, out: &Path) -> Result<(), Error> {
 
     let mut secret_key_bytes = [0u8; 32];
     OsRng.fill_bytes(&mut secret_key_bytes);
```

Comment:
Move this into `secret_key`?

Reply:
The data is read from a file In `read_keypair`. Only `generate_key` uses `OsRng.fill_bytes`.